### ISSUE TITLE
JWKS ToString broken on 1.9.1

### ISSUE THREAD

**Issue by watfordgnf** [uruk-project/Jwt#550]
We've begun serializing our JWKS to a well known location and found that under v1.9.1 it is broken. The fix was applied as part of a pre-release:
https://github.com/uruk-project/Jwt/commit/79695d7c62691769198a6d1c53814b4bbf86a02d#diff-d3b4dcfad73c68c10c1bc32d6fe3679468c657c1f05200384b59204959c0fbea

Is there a possibility of getting a v1.9.2 with just that targeted fix for JWKS? I'm willing to put together the PR and tests if that's a barrier.

**Comment by ycrumeyrolle** [comment]
Fixed in [1.9.2](https://www.nuget.org/packages/JsonWebToken/1.9.2). 
For your use case, I also change the visibility of the method `void WriteTo(Utf8JsonWriter writer)` to public. 

````C#
  // this code is not tested...
  using (var writer = new Utf8JsonWriter(context.Response.Body))
  {
    jwks.WriteTo(writer);
    writer.Flush();
  }
```

By doing so, you will avoid to allocate an UTF-16 string before to write to the response body

**Comment by watfordgnf** [comment]
I very much appreciate the rapid turnaround!
